build/zip_bundle.gni

```diff
@@ -38,16 +39,39 @@ if (flutter_runtime_mode == "jit_release") {
 template("zip_bundle") {
   assert(defined(invoker.output), "output must be defined")
   assert(defined(invoker.files), "files must be defined as a list of scopes")
+
+  license_readme = "$target_gen_dir/LICENSE.$target_name.md"
+  license_target = "_${target_name}_license_md"
+
+  generated_file(license_target) {
+    source_path = rebase_path(".", "//flutter")
+    license_path =
+        rebase_path("//flutter/sky/packages/sky_engine/LICENSE", "//flutter")
+    git_url = "https://github.com/flutter/engine/tree/$engine_version"
+    outputs = [ license_readme ]
+    contents = [
+      "# $target_name",
+      "",
+      "This bundle is part of the the Flutter SDK.",
+      "",
+      "The source code is hosted at [flutter/engine/$source_path]($git_url/$source_path).",
+      "The for this bundle is hosted at [flutter/engine/sky/packages/sky_engine/LICENSE]($git_url/$license_path).",
```

Comment:
is there any way we can make this URL point to a file on the same host as the zip files are hosted?

My concern is that the lifetime of github.com and the lifetime of the host where we upload the zip files is not guaranteed to be correlated, so if we suddenly find github goes away and we have to move all the code, all these old files will no longer point to a license file.

Reply:
We'd have to modify the recipes to do that, but we could. @godofredoc WDYT?

Reply:
Actually, I think we can now control it from the build json.. I'll look into that tomorrow.

Reply:
Ok, I've added a link to the sky_engine.zip location as well, which should have a full copy of the license.